lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -411,6 +416,19 @@ LogicalResult Serializer::processTypeDecoration<spirv::ArrayType>(
   }
   return success();
 }
+
+LogicalResult
+Serializer::processMemberDecoration(uint32_t structID, uint32_t memberNum,
```

Comment:
Can we name it as `memberIndex`? `memberNum` always makes me think we are passing in the total count here.

Reply:
Thanks, I'll update it!